handlers/LunchHandler.js

```diff
@@ -2,6 +2,16 @@ import Promise from 'bluebird';
 import BaseHandler from '../src/BaseHandler';
 import Facebook from '../src/Facebook';
 import moment from 'moment';
+import http from 'http';
+import cheerio from 'cheerio';
+import htmlToText from 'html-to-text';
+
+const optionsPaevapraed = {
```

Comment:
Avoid mixing estonian and english, code is always english (dailySpecialOptions).